Healthy/Classes/Modules/Splash/SplashViewController.swift

```diff
@@ -36,11 +36,12 @@ final class SplashViewController: UIViewController {
 
 // MARK: - Actions
 
-private extension SplashViewController {
+extension SplashViewController {
     @IBAction func didTapStartCooking(_ sender: Any) {
-        let viewModel = LoginViewModel()
-        let viewController = LoginViewController(viewModel: viewModel)
+        let loginviewModel = LoginViewModel()
```

Comment:
Please revert the naming here. 
```suggestion
        let viewModel = LoginViewModel()
```

Reply:
I have another variable called viewModel for a splash

Reply:
But we need only one in the scope of the method. You can still access the outer view model with `self.viewModel`